### DISCUSSION TITLE
wxTimer management by using wxUiEditor

### DISCUSSION THREAD

**Discussion by rossanoparis** [KeyWorksRW/wxUiEditor#1268] [General]
I have read the issue #343, despite this is there any chance to change your mind regarding wxTimer ?
It is a class like any one in wx, and it could be useful to manage it using the RAD tool

**Comment by Randalphwa** [comment]
You've been incredibly helpful in locating and describing bugs, as well as suggesting features. It seems like the least I can do is add support for wxTimer... 😄

PR #1283 adds support for wxTimer, including importing it from wxFormBuilder (and wxCrafter).  It should be in the 11/9 daily build.

**Comment by rossanoparis** [comment]
Thank you @Randalphwa the main reason is that I like your project very much and even more I like your attitude to listen to suggestions and so on, if you like the way I use to report bugs and other stuffs I'd be very pleased to help you improving wxUiEditor in that way ...
I'm very keen to get rid of other RAD tools I've been using since the last 20 years, so I started to import projects realized using wxFormBuilder (I have 45 projects tgo import)
After that, I have 53 huge projects built using DialogBlocks (which was a very clver and smart RAD tool, but aout of date), this will be a hard work for me, butg I'm confident to get my goal thanks to wxUiEditor.

Long life and prosperity :)
